server/src/com/cloud/storage/upload/UploadListener.java

```diff
@@ -113,7 +113,7 @@ protected void runInContext() {
 
     private DataStore sserver;
 
-    private boolean uploadActive = true;
+    // private boolean uploadActive = true;
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable uploadActive was only written to (never read from). Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this variable.